pycaret/internal/pycaret_experiment/tabular_experiment.py

```diff
@@ -650,7 +650,7 @@ def pipeline():
 
                     # Create schematic drawing
                     d = Drawing(backend="matplotlib")
-                    d.config(fontsize=12)
+                    d.config(fontsize=plot_kwargs.get("fontsize", 14))
```

Comment:
@tvdboom Can you update the docstring to show that it can accept this key ("fontsize") now? Thanks!